epub33/core/index.html

```diff
@@ -5320,7 +5320,7 @@ <h4>Layout</h4>
 
 					</div>
 
-					<p>When the property is set to <code>pre-paginated</code> for a spine item, its content dimensions
+					<p id="fxl-layout-duplication" data-tests="fxl-layout_duplication">When the property is set to <code>pre-paginated</code> for a spine item, its content dimensions
```

Comment:
Hyphen instead of underscore in the data-tests. Here's the pattern:

id="foo-bar" data-tests="foo-bar"
id="foo-bar" data-tests="foo-bar_baz,foo-bar_qux"

That is, underscores only if you need multiple tests for the same statement; the underscores are to differentiate them.

Reply:
Also all data-tests need to start with a #, please, because they refer to anchors in the test report.